### ISSUE TITLE
Plugin does not load on iOS

### ISSUE THREAD

**Issue by velebit** [zlovatt/obsidian-trim-whitespace#14]
When I try to enable this plugin in the iOS version of Obsidian, I get the error message: `Failed to load plugin obsidian-trim-whitespace`

**Comment by zlovatt** [comment]
I have no access to any ios devices; if there's any way to view info on what's failing, that would be handy!

**Comment by velebit** [comment]
I'm planning to take a look but it might take a bit of time before I get there. The bad news is that viewing the console data is not as straightforward as on the desktop, but the good news is that it should be possible if one creates a separate simple plugin to save the log to a file in the vault, as described here: https://gist.github.com/liamcain/3f21f1ee820cb30f18050d2f3ad85f3f

**Comment by velebit** [comment]
The error is:
```
[error] Plugin failure: obsidian-trim-whitespace SyntaxError: Invalid regular expression: invalid group specifier name
```
which is probably related to this:
https://marcus.se.net/obsidian-plugin-docs/testing/mobile-devices#lookbehind-in-regular-expressions
but I haven't had a chance to investigate further.

**Comment by velebit** [comment]
As a test, I removed the look-behind assertions from the regular expressions in trimText.ts, and the plugin started loading on iOS. So it looks like that's the problem.

@zlovatt Any thoughts on the way forward?

**Comment by velebit** [comment]
See PR #15.

**Comment by velebit** [comment]
Can confirm the bug is fixed in 0.3.0 on iOS.
